src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -0,0 +1,354 @@
+#    Copyright 2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+"""Implementation of PCR tests."""
+import json
+import logging
+import os
+import signal
+import time
+import subprocess
+import sas_testcase
+from shapely import ops
+from full_activity_dump_helper import getFullActivityDumpSasUut
+from reference_models.ppa import ppa
+from reference_models.geo import drive, utils
+from util import configurable_testcase, loadConfig, \
+     makePalRecordsConsistent, writeConfig, getCertificateFingerprint
+TEST_CONFIG_COMMAND = 'cd reference_models;python test_config.py'
```

Comment:
Please remove this. This script shall not be used at all..

Reply:
Due to   previous comment this line has been no longer required . So removed